<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiClockApp</class>
 <widget class="QMainWindow" name="PiClockApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PiClockApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="layout">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="start">
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>129</y>
        <width>800</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tlwg Typist</family>
        <pointsize>120</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PiClock</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>265</y>
        <width>740</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="main">
     <widget class="QWidget" name="clock" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="time">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>10</y>
         <width>395</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tlwg Typo</family>
         <pointsize>60</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
      <widget class="QLabel" name="date">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>390</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="widgetSlideshow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>780</width>
        <height>310</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="nw">
       <widget class="QLabel" name="nw_title">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>780</width>
          <height>310</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>60</pointsize>
         </font>
        </property>
        <property name="text">
         <string>no widgets enabled</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="settingsButton">
      <property name="geometry">
       <rect>
        <x>726</x>
        <y>10</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <widget class="QPushButton" name="mainButton">
      <property name="geometry">
       <rect>
        <x>726</x>
        <y>10</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTabWidget" name="options">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>780</width>
        <height>350</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="general">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QComboBox" name="clockTimeFormatOptions">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>55</y>
          <width>300</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>12 Hour (5:00pm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24 Hour (17:00)</string>
         </property>
        </item>
       </widget>
       <widget class="QSlider" name="slideDurationSlider">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>55</y>
          <width>420</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>0</number>
        </property>
        <property name="pageStep">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="clockTimeFormatLabel">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>15</y>
          <width>300</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Clock Time Format</string>
        </property>
       </widget>
       <widget class="QLabel" name="slideDurationLabel">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>15</y>
          <width>250</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Widget Slide Duration</string>
        </property>
       </widget>
       <widget class="QLabel" name="slideDurationValueLabel">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>15</y>
          <width>170</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>(seconds)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="alarms">
       <attribute name="title">
        <string>Alarms</string>
       </attribute>
       <widget class="QComboBox" name="alarmGameEdit">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>50</y>
          <width>200</width>
          <height>45</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QComboBox" name="alarmDifficultyEdit">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>150</y>
          <width>200</width>
          <height>45</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="alarmSetButton">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>245</y>
          <width>200</width>
          <height>45</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
       <widget class="DragListWidget" name="alarmList">
        <property name="geometry">
         <rect>
          <x>421</x>
          <y>0</y>
          <width>355</width>
          <height>312</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="DragListWidget" name="alarmTimeOptions">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>200</width>
          <height>262</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QLabel" name="alarmTimeLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Time</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="alarmGameLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Game</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="alarmDifficultyLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>105</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Difficulty</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widgets">
       <attribute name="title">
        <string>Widgets</string>
       </attribute>
       <widget class="QCheckBox" name="wclockCheckbox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>142</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="forecastCheckbox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>82</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="enabledLabel">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>15</y>
          <width>125</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Is Enabled?</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="worldClocksCheckbox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>142</y>
          <width>211</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="holidayCountdownCheckbox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>172</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="weatherCheckbox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>52</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="quotesCheckbox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>112</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>46</y>
          <width>20</width>
          <height>197</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>62</y>
          <width>25</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>92</y>
          <width>25</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>122</y>
          <width>25</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_5">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>152</y>
          <width>25</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_6">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>182</y>
          <width>25</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_7">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>212</y>
          <width>25</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_8">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>242</y>
          <width>25</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Weather</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Forecast</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Quotes</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>140</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>World Clocks</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>170</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Holiday Countdown</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>200</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>System Usage</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>230</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Commute</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="sysstatsCheckbox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>202</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="commuteCheckbox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>232</y>
          <width>200</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="settingsHeader">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>250</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tlwg Typo</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SETTINGS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsSaveButton">
      <property name="geometry">
       <rect>
        <x>665</x>
        <y>430</y>
        <width>125</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tlwg Typo</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsCancelButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>430</y>
        <width>125</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tlwg Typo</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="alarm">
     <widget class="QStackedWidget" name="games">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>500</width>
        <height>410</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pointsRequiredLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>280</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Points Required</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="currentPointsLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>280</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>90</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="timeRemainingLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>280</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Time Remaining</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="currentTimeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>280</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>90</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="text">
       <string>4:59</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DragListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">settingsPage/draglistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
